srcpkgs/wireproxy/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'wireproxy'
+pkgname=wireproxy
+version=1.0.1
+revision=1
+build_style=go
+go_import_path="github.com/octeep/wireproxy"
+go_package="${go_import_path}/cmd/wireproxy"
+short_desc="Wireguard client that exposes itself as a socks5 proxy"
+maintainer="Wind Wong <octeep@pm.me>"
+license="ISC"
+homepage="https://github.com/octeep/wireproxy"
+distfiles="https://github.com/octeep/wireproxy/archive/v${version}.tar.gz"
+checksum=e21eac22ef1b12dc2d7b3e5b58dcff183af9c1547be3e55a7e6c71394f053e1e
+
+system_accounts="_wireproxy"
+_wireproxy_homedir="/var/lib/wireproxy"
+
+make_dirs="
+	/etc/wireproxy 0750 root _wireproxy
+	/var/lib/wireproxy 0700 _wireproxy _wireproxy"
```

Comment:
```suggestion
make_dirs="
 /etc/wireproxy 0750 root _wireproxy
 /var/lib/wireproxy 0700 _wireproxy _wireproxy"
```
and $make_dirs can be in the main block after $checksum

Reply:
It's not really visible, but there should be a single space character on start of each line in ""